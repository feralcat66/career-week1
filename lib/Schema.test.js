const {
  newFunction
} = require('./Schema.js');

describe('Schema', () => {
    const nameValidator = new ('name', {
      type: String,
      required: true
    });
    

const blobbly = {
    name: 'blobbly',
    age: 30,
    height: '6 ft',
    eyeColor: 'brown'
  };


  
describe('Schema', () => {
  it('tests for proper configuration of an object', () => {
   
  });
  
});

it('throws if format is unacceptable', () => {
  expect(() => nameValidator(1234)).toThrowErrorMatchingSnapshot();
});
  
expect(func).toEqual('name');