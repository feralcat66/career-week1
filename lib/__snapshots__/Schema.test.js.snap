// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`throws if format is unacceptable 1`] = `"nameValidator is not defined"`;
